cmd/mev-boost/main.go

```diff
@@ -37,7 +37,7 @@ func main() {
 		_relayURLs = append(_relayURLs, strings.Trim(entry, " "))
 	}
 
-	store := lib.NewStore()
+	store := lib.NewStore(true)
```

Comment:
```suggestion
	store := lib.NewStore(true /* start clean up loop /*)
```

Reply:
Rather than passing in boolean flag, another approach is:

```go
func NewStore() Store {
  return s
}

func NewStoreWithCleanup() Store {
  s := NewStore()
  // Start clean up routine
  return s
}
```

Reply:
I like it!

Reply:
updated